src/App.js

```diff
@@ -20,7 +23,11 @@ function App() {
     }
   }, [i18n, language]);
 
-  return <Routes />;
+  return (
+    <ReactKeycloakProvider authClient={keycloak}>
```

Comment:
![Screen Shot 2021-01-18 at 11 45 55 AM](https://user-images.githubusercontent.com/26819992/104942654-bad3df00-5982-11eb-943f-a0d1dc8a754d.png)

See documentation. Any other providers should be within this one. recommend adding it to the index.js instead

Reply:
ok